style.css

```diff
@@ -89,4 +89,209 @@
     width: 170px;
     top: 536px;
   }
+
+  .mid-section {
+    display: grid;
+    grid-template-columns: 327px;
+    grid-template-rows: auto auto auto auto auto;
+    grid-gap: 15px;
+    justify-content: center;
+  }
+
+  .intro {
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
+    align-items: center;
+  }
+
+  .intro h2 {
+    margin: 0;
+  }
+
+  #indicator {
+    height: 4px;
+    width: 48px;
+    top: 121px;
+    border-radius: 24px;
+    background-color: #36b37e;
+  }
+
+  .work-section {
+    display: grid;
+    grid-template-columns: 327px;
+    grid-template-rows: 474px 474px 474px;
+    grid-gap: 24px;
+    align-items: center;
+  }
+
+  .entries {
+    display: flex;
+    flex-direction: column;
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+  }
+
+  .entries img {
+    border-radius: 8px 8px 0 0;
+  }
+
+  .description {
+    background: #fff;
+    align-items: center;
+    border-radius: 0 0 8px 8px;
+  }
+
+  .description h4 {
+    font-size: 24px;
+    font-weight: 600;
+    line-height: 32px;
+    letter-spacing: 0;
+    text-align: center;
+    padding: 0;
+    margin: 10px;
+  }
+
+  .description ul {
+    margin: 0, 11px;
+    display: flex;
+    flex-grow: 1;
+    justify-items: space-between;
+  }
+
+  .list1 li {
+    background: #ebf0ee;
+    border-radius: 4px;
+    margin: 0 5px 0 5px;
+    padding: 5px;
+    font-size: 12px;
+    font-weight: 600;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    text-align: left;
+  }
+}
+
+.button-1 {
+  color: white;
+  height: auto;
+  width: 98px;
+  left: 12px;
+  top: 12px;
+  margin-left: 103px;
+  margin-bottom: 10px;
+  border-radius: 4px;
+  background: #36b37f;
```

Comment:
- [ ] Kindly make sure your button styles look as the ones in Figma. And you may also consider adding the interaction style such as `hover`, `active`, and `disable`
- [ ] Also consider readjusting the color of your elements
`Hint: To get the element style, just click it and on the right you will be able to see the styles applied to it in Figma.` :smile: 
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205437643-08d2e3fb-2d35-4b53-8c35-fa879ec793ea.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205437660-391543f9-f284-4058-8fcd-85dda1e7cc1d.png)